rpc_jobs/standard_job_postmerge.yml

```diff
@@ -53,6 +55,7 @@
       env.RE_JOB_ACTION = "{action}"
       env.RE_JOB_FLAVOR = "{FLAVOR}"
       env.RE_JOB_TRIGGER = "PM"
+      env.RE_JOB_REPO_NAME = "{repo_name}"
```

Comment:
This should be documented in RE-For-Projects, or not injected into the environment (remove env.)